### PR TITLE
Main Library Functionality of useGlobalState

### PR DESCRIPTION
**Pull request by r17x**
Related at https://github.com/evilfactorylabs/useGlobalState/issues/1 

Make the main Library functionality base on React.Hook + Context.

### CODE REVIEW

**Comment by faultables** [comment]
@ri7nz have any advertising material related with this, maybe like "Why use this instead use X?

**Comment by r17x** [comment]
@108kb  maybe, we can add-in release.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,103 @@
...
+Example Use of `<StateProvider/>`.
+
+
+```javascript
+import React, {useReducer} from 'react'
+import App from './you-app.js'
+import {StateProvider} from 'evilfactory/global-state'
+
+const initialState = { todo: [] } 
+const reducer = useReducer(state, action)
```
Comment (faultables) [inline README.md]: > const reducer = useReducer(state, action)

Does it means:

```diff
- const reducer = useReducer(state, action)
+ const reducer = useReducer(initialState, action)
```

right?

**Review by faultables — COMMENTED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (r17x) [reply README.md]: done @108kb

**Review by faultables — APPROVED** [review]

**Pull request merged**